### PR TITLE
fix: Update OpenStack presentation, utilisation and deploiement modules

### PR DESCRIPTION
**Pull request by jftalta**
A few more OpenSack course updates.

### CODE REVIEW

**Comment by jftalta** [comment]
Hello,

Is there something that blocks this PR ?

File: cours/openstack/openstack-presentation.fr.md [hunk 1]
```diff
@@ -18,14 +18,14 @@
 
 ### Mission
 
-"To produce an ubiquitous Open Source Cloud Computing platform that is
+```To produce an ubiquitous Open Source Cloud Computing platform that is
```
Comment (rguichard) [inline cours/openstack/openstack-presentation.fr.md:21]: Block \```code``` shouldn't be used here. It displays the text on a single line. `"` Should be used instead imo.

File: cours/openstack/openstack-presentation.fr.md [hunk 2]
```diff
@@ -18,14 +18,14 @@
...
 easy to use, simple to implement, interoperable between deployments,
 works well at all scales, and meets the needs of users and operators of
-both public and private clouds."
+both public and private clouds.```
 
 ### OpenStack en 2020
 
-"OpenStack is one of the top 3 most active open source projects and manages 10 million compute cores"
+```OpenStack is one of the top 3 most active open source projects and manages 10 million compute cores```
```
Comment (rguichard) [inline cours/openstack/openstack-presentation.fr.md:28]: same as above

File: cours/openstack/openstack-presentation.fr.md [hunk 3]
```diff
@@ -118,23 +118,29 @@ Autres :
...
 ![Les principales entités de la Fondation](images/foundation.png)
 
 ### Open Infrastructure
 
--   En 2018, la Fondation s'élargit à l'**Open Infrastructure**
+-   En 2018, la Fondation OpenStack s'élargit à l'**Open Infrastructure**
+
+<https://openinfra.dev/about/open-infrastructure/>
+
+```
```
Comment (rguichard) [inline cours/openstack/openstack-presentation.fr.md:139]: same as above

**Review by rguichard — CHANGES_REQUESTED** [review]
Just a few things concerning display. \``` for anything else than code end up pretty ugly once the course is built.

**Comment by rguichard** [comment]
Sorry for the delay, forgot to do the review sooner ^^

File: cours/openstack/openstack-presentation.fr.md (hunk 1 above)
Comment (jftalta) [reply cours/openstack/openstack-presentation.fr.md:21]: I agree.

File: cours/openstack/openstack-presentation.fr.md (hunk 2 above)
Comment (jftalta) [reply cours/openstack/openstack-presentation.fr.md:28]: I agree.

File: cours/openstack/openstack-presentation.fr.md (hunk 3 above)
Comment (jftalta) [reply cours/openstack/openstack-presentation.fr.md:139]: I agree.

**Comment by jftalta** [comment]
No problem with the delay :-)

**Review by rguichard — APPROVED** [review]
lgtm

**Pull request merged**